notes.go

```diff
@@ -73,6 +73,8 @@ type Note struct {
 }
 
 // NotePosition represents the position attributes of a note.
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/discussions.html#create-new-merge-request-thread
```

Comment:
Lets omit these lines as this struct is not used only for the API call.

Reply:
👌